src/OrchardCore.Modules/OrchardCore.OpenId/Views/Access/Authorize.cshtml

```diff
@@ -1,10 +1,14 @@
-﻿@using OrchardCore.OpenId.ViewModels
+@using OrchardCore.OpenId.ViewModels
 @model AuthorizeViewModel
 
+@{
+    ViewLayout = "Layout__Login";
```

Comment:
Could we move it to `_ViewStart.cshtml`?

Reply:
It's `Layout` or `ViewLayout`?!!

Reply:
It is a trick to use the layout from front end theme instead of the admin. It is used on all login views

Reply:
I see